src/main/java/BasicJavaHomeWork/Exercise_4.java

```diff
@@ -0,0 +1,41 @@
+package BasicJavaHomeWork;
+
+import java.util.Scanner;
+
+public class Exercise_4 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Please enter a number: ");
+        int numb = in.nextInt();
+        int b;
+        int c = 0;
+
+        for (int a=1; a<=numb;a++) {
+            b = a + c;
+            c = b;
```

Comment:
this can be simplified to use less variables

and next methods as well

Reply:
done